Applications/FileManager/DirectoryView.cpp

```diff
@@ -46,6 +46,8 @@ void DirectoryView::handle_activation(const GModelIndex& index)
 DirectoryView::DirectoryView(GWidget* parent)
     : GStackWidget(parent)
     , m_model(GDirectoryModel::create())
+    , path_history_position(0)
```

Comment:
Please initialize this in the header instead;
`int path_history_position { 0 };`